flight/targets/coptercontrol/fw/pios_board.c

```diff
@@ -140,6 +140,10 @@ void PIOS_Board_Init(void) {
 	/* Mount all filesystems */
 	PIOS_FLASHFS_Logfs_Init(&pios_uavo_settings_fs_id, &flashfs_settings_cfg, FLASH_PARTITION_LABEL_SETTINGS);
 
+#if defined(ERASE_FLASH)
+	PIOS_FLASHFS_Format(pios_uavo_settings_fs_id);
+#endif
+
```

Comment:
I'd really like to avoid pios_boards further diverging because i'd like to work on factoring them common.  That being said, if you really object I'll make it so CC3D (but none of our other targets) pause after flash erase.